### DISCUSSION TITLE
when I migrate the chia data from one server to another server, the wallet can not open

### DISCUSSION THREAD

**Discussion by blocknodehub** [Chia-Network/chia-blockchain#11727] [Support]
when I migrate the chia data from one server to another server, the wallet can not open
the two servers have same OS and data directory
when I start node and wallet, it prompt as blew
```
chia wallet show
No keys loaded. Run 'chia keys generate' or import a key
```

actually I can find my private key

**Comment by loppefaaret** [comment]
add your key with `chia keys add`
